config/site_settings.yml

```diff
@@ -1322,6 +1322,11 @@ files:
     type: list
     list_type: simple
     default: ""
+  block_hotlinked_media:
+    default: false
+  block_hotlinked_media_exceptions:
```

Comment:
I'm wondering if some basic validations of the URLs that is set here. Having valid URLs stored here will make it easier down the road to work with the site settings.

Reply:
Re-reading this again, it seems like the protocol needs to be included too so I think a validator for this site setting would be very helpful.